<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recordingOptions</class>
 <widget class="QDialog" name="recordingOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recording Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="recordDurationR">
       <property name="toolTip">
        <string>Record stream for specified duration in seconds</string>
       </property>
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="durationSpin">
       <property name="toolTip">
        <string>Record stream for specified duration in seconds</string>
       </property>
       <property name="maximum">
        <double>3600.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>60.000000000000000</double>
       </property>
       <property name="value">
        <double>30.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Record:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Set SER video &lt;b&gt;remote&lt;/b&gt; directory name. If INDI server is running locally, then a local directory can be selected. However, if you are connected to a remote INDI server, then the directory must be a valid directory on the remote file system where it is saved.&lt;/p&gt;
&lt;p&gt;Record directories may contain some patterns to make them dynamic:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;&lt;b&gt;_D_&lt;/b&gt; for the date in YYYY-MM-DD.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_H_&lt;/b&gt; for time in HH:MM:SS.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_T_&lt;/b&gt; for ISO8601 time stamp.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_F_&lt;/b&gt; for filter name, if any.&lt;/li&gt;
&lt;/ul&gt;
&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Directory:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLineEdit" name="recordDirectoryEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Set SER video &lt;b&gt;remote&lt;/b&gt; directory name. If INDI server is running locally, then a local directory can be selected. However, if you are connected to a remote INDI server, then the directory must be a valid directory on the remote file system where it is saved.&lt;/p&gt;
&lt;p&gt;Record directories may contain some patterns to make them dynamic:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;&lt;b&gt;_D_&lt;/b&gt; for the date in YYYY-MM-DD.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_H_&lt;/b&gt; for time in HH:MM:SS.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_T_&lt;/b&gt; for ISO8601 time stamp.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_F_&lt;/b&gt; for filter name, if any.&lt;/li&gt;
&lt;/ul&gt;
&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectDirB">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Play/Pause</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="recordFilenameEdit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Set SER video file name. Record files may contain some patterns to make them dynamic:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;&lt;b&gt;_D_&lt;/b&gt; for the date in YYYY-MM-DD.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_H_&lt;/b&gt; for time in HH:MM:SS.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_T_&lt;/b&gt; for ISO8601 time stamp.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_F_&lt;/b&gt; for filter name, if any.&lt;/li&gt;
&lt;/ul&gt;
&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="recordFramesR">
       <property name="toolTip">
        <string>Record stream until this many frames are captured</string>
       </property>
       <property name="text">
        <string>Frames</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="framesSpin">
       <property name="toolTip">
        <string>Record stream until this many frames are captured</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="recordUntilStoppedR">
     <property name="toolTip">
      <string>Record stream until manually stopped</string>
     </property>
     <property name="text">
      <string>Until Stopped</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Set SER video file name. Record files may contain some patterns to make them dynamic:&lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;&lt;b&gt;_D_&lt;/b&gt; for the date in YYYY-MM-DD.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_H_&lt;/b&gt; for time in HH:MM:SS.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_T_&lt;/b&gt; for ISO8601 time stamp.&lt;/li&gt;
&lt;li&gt;&lt;b&gt;_F_&lt;/b&gt; for filter name, if any.&lt;/li&gt;
&lt;/ul&gt;
&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>File name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>recordingOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>recordingOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
